js/base-chat-languages.js

```diff
@@ -0,0 +1,2427 @@
+function baseChatLanguages () {
```

Comment:
Yep I thought a lot about JSON, and yes it happens that base-emoji (which existed before my edits) is also JSON. I even thought about consulting with you guys, but decided to let it be as it is for now. I still hope we will get to a major restructuring and do something about it.

Reply:
Btw I think probably config descriptions may also be done as JSON. Ofc it would not be available to put config parts to the .js files they're related to. On the other hand, maybe it would be a good thing, as now it's sometimes hard to find particular option declaration since they're all scattered across js.